utils/ultralytics/train.py

```diff
@@ -69,6 +73,7 @@ def ultralytics_train_yolov5(
             "kili.yaml",
             "--project",
             f"{output_path}",
+            "--upload_dataset",  # wandb
```

Comment:
I would add a `--name` argument to set a better name in W&B (now it is something like 2022-04-12 11_43_39). I would be in favor of "Kili project name _ job name", for example "Severstal-steel-defect-detection_DETECTION"

Reply:
The name in wandb depends on from the --project argument. I've added the title of the project in the name

Reply:
can you also add the job_name ad the end of the name? there can be several jobs per project.